external/vcm/vcm/calc/thermo.py

```diff
@@ -396,22 +399,25 @@ def liquid_ice_temperature(
 
 
 def column_integrated_heat(
-    temperature: xr.DataArray, delp: xr.DataArray, vertical_dim: str = "z"
+    temperature: xr.DataArray,
+    delp: xr.DataArray,
+    isochoric: bool = True,
```

Comment:
I think this function is ambiguous and could lead to more confusion later on. 

- "heat" is not very specific.
- "isochoric" refers to holding volume constant while the other phase variables change, but this function does not involve any thermodynamic transitions. It is a function of state.

I suggest making two functions: `column_enthalpy` and/or `column_internal_energy` to disambiguate.

Reply:
I'll probably just undo the changes (or delete this function) since it's not used anywhere in our codebase as far as I can tell. The only difference between this function and `column_integrated_heating` is the attributes attached to the output array.